src/algebra/opposites.lean

```diff
@@ -192,6 +193,20 @@ variable {α}
 @[simp] lemma unop_smul {R : Type*} [has_scalar R α] (c : R) (a : αᵒᵖ) :
   unop (c • a) = c • unop a := rfl
 
+lemma semiconj_by.op [has_mul α] {a x y : α} (h : semiconj_by a x y) :
```

Comment:
Can you add the statement about `unop` too?

Reply:
Thanks for the suggestion. I did.